package.json

```diff
@@ -1,87 +0,0 @@
-{
```

Comment:
Please, revert deleting the `package.json`

Reply:
fixed